### DISCUSSION TITLE
Bases displayed as quote on 2022.02.14.1 (maybe RU only issue)

### DISCUSSION THREAD

**Discussion by MarshalX** [pradyunsg/furo#378] [Potential Issue] [answered]
Displayed as quote: https://yandex-music.readthedocs.io/ru/dev/yandex_music.base.html#yandex_music.base.YandexMusicObject
Normal displaying: https://wrtc.readthedocs.io/en/latest/webrtc.base.html#webrtc.base.WebRTCObject

<img width="859" alt="obraz" src="https://user-images.githubusercontent.com/15520314/154293587-0cf16d18-2519-4988-91be-9b491319f02d.png">

<img width="795" alt="obraz" src="https://user-images.githubusercontent.com/15520314/154293530-c09ab7f3-1879-41ab-8207-40e5efd59df6.png">

**Comment by pradyunsg** [comment, marked as answer]
Hmm... I'm not sure this is a theme-level issue -- Furo does not interfere with the markup generation for this stuff. Can you make a minimal reproducer for this for confirming whether this is a Sphinx bug?

The HTML for that bit has a blockquote:

```
<blockquote>
<div><p>Базовые классы: <code class="xref py py-class docutils literal notranslate"><span class="pre">object</span></code></p>
</div></blockquote>
```

Furo is one of the few Sphinx themes that renders blockquotes visibly tho, which is why this can be spotted in it. So... that's good? 🤷🏽‍♂️

**Reply by dgw** [reply]
Yep. When we moved to Furo it put a spotlight on [incorrect markup](https://github.com/sopel-irc/sopel/commit/bd1af5e04080b85cffca5407270c93c68c2a5200) that had sat there unnoticed for literal years. (Thanks for making it obvious so we could fix it, btw!)

**Reply by MarshalX** [reply]
Thanks for the answer! You are right. I make a pull request to Sphinx: https://github.com/sphinx-doc/sphinx/pull/10203

**Reply by MarshalX** [reply]
@pradyunsg btw any news about this https://github.com/pradyunsg/furo/discussions/371 ?)
